packages/block-library/src/button/save.js

```diff
@@ -60,7 +60,8 @@ export default function save( { attributes, className } ) {
 
 	return (
 		<div { ...useBlockProps.save( { className: wrapperClasses } ) }>
-			<RichText.Content
+			<BlockButton
```

Comment:
This is going to require a deprecation for all blocks that uses it in save functions because of the extra classname.

Reply:
It seems to cope OK without a deprecation - I see this message in the console:

```
Block successfully updated for `core/button` ({name: 'core/button', icon: {…}, keywords: Array(1), attributes: {…}, providesContext: {…}, …}).

New content generated by `save` function:

<div class="wp-block-button"><a class="wp-block-button__link wp-element-button">button 2</a></div>

Content retrieved from post body:

<div class="wp-block-button"><a class="wp-block-button__link">button 2</a></div>

```

Is it still better to provide a deprecation?

Reply:
I think you got lucky that an existing deprecation handled this for you. If your block included a custom font family, the old deprecation wouldn't have worked because the markup would be unrecognized by the old deprecation as well. 

The rule is simple any change to the save function (saved markup) requires a deprecation.

Reply:
I tried with a custom font family, but I still didn't get an error. Any clues how I can trigger one?

I have added a deprecation but it doesn't have `isEligable` or `migrate` properties as I'm not sure what it is I need to be migrating from and to.

Reply:
I have tested my deprecation and it seems to work correctly, but I do get this message in the console, is that expected?

```
Block validation: Block validation failed for `core/button` ({name: 'core/button', icon: {…}, keywords: Array(1), attributes: {…}, providesContext: {…}, …}).

Content generated by `save` function:

<div class="wp-block-button is-style-outline"><a class="wp-block-button__link has-text-color has-background wp-element-button" style="background-color:#000000;color:#ffffff">Visit</a></div>

Content retrieved from post body:

<div class="wp-block-button is-style-outline"><a class="wp-block-button__link has-text-color has-background no-border-radius" style="background-color:#000000;color:#ffffff">Visit</a></div>
```